compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIETileAndFuse.cpp

```diff
@@ -128,9 +154,14 @@ void AMDAIETileAndFusePass::runOnOperation() {
   // tiling strategy.
   // If `consumerOp` has its own lowering config, we prefer using it.
   // Otherwise, fallback to find a lowering_config from other operations.
+  // When the consumerOp is a linalg.copy, the lowering config is originally
+  // added to the matmul op. To tile the linalg.copy op, take lowering configs
+  // from the its user matmul.
   SmallVector<int64_t> tileSizesVal;
   if (auto loweringConfig = getLoweringConfig(consumerOp)) {
     tileSizesVal = loweringConfig.getTileSizeVals(tilingLevel);
+  } else if (isa<linalg::CopyOp>(consumerOp) && getLoweringConfig(matmulOp)) {
+    tileSizesVal = getLoweringConfig(matmulOp).getTileSizeVals(tilingLevel);
```

Comment:
I guess the following should work :-

```
if (isa<linalg::CopyOp>(consumerOp) && ( auto loweringConfig = getLoweringConfig(matmulOp)))
```

Just to reuse the loweringConfig and not re-invoke the same function.

Reply:
It doesn't work in this way.